server.tf

```
@@ -14,6 +14,7 @@ data "template_file" "nessus-user-data" {
 # Find the latest AMI by product code
 data "aws_ami" "nessus-image" {
   most_recent = true
+  owners = ["679593333241"]
```

Looking at the [aws_ami data source](https://www.terraform.io/docs/providers/aws/d/ami.html#owners), it appears to be possible to use a friendly name for AWS Marketplace AMIs instead of that ID.  Would you mind checking to see if that works instead, then update the PR and I'll merge?

Thanks!

```suggestion
  owners = ["aws-marketplace"]
```

---

updated with the friendly name, that worked.